src/main/java/core/basesyntax/ArrayListIndexOutOfBoundsException.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public class ArrayListIndexOutOfBoundsException extends RuntimeException {
-    public ArrayListIndexOutOfBoundsException(String message) {
-        super(message);
+    public ArrayListIndexOutOfBoundsException(int index, int size) {
+        super("Index: " + index + "; size: " + size);
```

Comment:
You shouldn't make changes to this class, why limit yourself to only one exception message